lang/sql/sql.talon

```diff
@@ -1,20 +1,47 @@
-app: datagrip
-
+tag: user.sql
 -
-select: "SELECT "
-star: "*"
+tag(): user.code_operators_math
+tag(): user.code_comment_line
+tag(): user.code_comment_block_c_like
+tag(): user.code_data_null
+tag(): user.code_functions_common
+
+^select$: "SELECT "
+distinct: "DISTINCT "
 from: "FROM "
-select star from: "SELECT * FROM "
+^select star from$: "SELECT *\nFROM "
 where: "WHERE "
 order by: "ORDER BY "
+group by: "GROUP BY "
+having: "HAVING "
 descending: " DESC"
 ascending: " ASC"
 dot i d: ".id"
-is not null: " IS NOT NULL"
-is null: " IS NULL"
-inner join:
-    insert("INNER JOIN  ON ")
-    key(left)
-    key(left)
-    key(left)
-    key(left)
+inner join: user.insert_between("INNER JOIN ", " ON ")
+left outer join: user.insert_between("LEFT OUTER JOIN ", " ON ")
+right outer join: user.insert_between("RIGHT OUTER JOIN ", " ON ")
```

Comment:
I still have an objection to inserting `ON`, since PostgreSQL has equi-join with `USING`.